src/main/java/com/amazon/ion/impl/bin/IonRawBinaryWriter_1_1.kt

```diff
@@ -345,71 +344,37 @@ class IonRawBinaryWriter_1_1 internal constructor(
         currentContainer.usesFlexSym = true
     }
 
-    override fun writeNull() = writeScalar { IonEncoder_1_1.writeNullValue(buffer, IonType.NULL) }
+    override fun writeNull() = writeScalar { writeNullValue(buffer, IonType.NULL) }
 
-    override fun writeNull(type: IonType) = writeScalar { IonEncoder_1_1.writeNullValue(buffer, type) }
+    override fun writeNull(type: IonType) = writeScalar { writeNullValue(buffer, type) }
 
-    override fun writeBool(value: Boolean) = writeScalar { IonEncoder_1_1.writeBoolValue(buffer, value) }
+    override fun writeBool(value: Boolean) = writeScalar { writeBoolValue(buffer, value) }
 
-    override fun writeInt(value: Byte) {
-        TODO("Not yet implemented")
-    }
+    override fun writeInt(value: Long) = writeScalar { writeIntValue(buffer, value) }
 
-    override fun writeInt(value: Int) {
-        TODO("Not yet implemented")
-    }
+    override fun writeInt(value: BigInteger) = writeScalar { writeIntValue(buffer, value) }
 
-    override fun writeInt(value: Long) {
-        TODO("Not yet implemented")
-    }
+    override fun writeFloat(value: Float) = writeScalar { writeFloatValue(buffer, value) }
 
-    override fun writeInt(value: BigInteger) {
-        TODO("Not yet implemented")
-    }
+    override fun writeFloat(value: Double) = writeScalar { writeFloatValue(buffer, value) }
 
-    override fun writeFloat(value: Float) {
-        TODO("Not yet implemented")
-    }
+    override fun writeDecimal(value: BigDecimal) = writeScalar { writeDecimalValue(buffer, value) }
 
-    override fun writeFloat(value: Double) {
-        TODO("Not yet implemented")
-    }
-
-    override fun writeDecimal(value: BigDecimal) {
-        TODO("Not yet implemented")
-    }
-
-    override fun writeDecimal(value: Decimal) {
-        TODO("Not yet implemented")
-    }
-
-    override fun writeTimestamp(value: Timestamp) {
-        TODO("Not yet implemented")
-    }
+    override fun writeTimestamp(value: Timestamp) = writeScalar { writeTimestampValue(buffer, value) }
 
     override fun writeTimestamp(value: Instant) {
         TODO("Not yet implemented")
```

Comment:
🗺️ I started to implement this, but I realized how expensive it is to convert an `Instant` into its year, month, day, etc., and so I don't think it offers much benefit over just converting the `Instant` to a `Timestamp`. As long as there's no objection, I think I'll just remove it.

It might be useful to have streamlined function for `LocalDate` and it's ilk at some point in the future.